### PR TITLE
Fix for logger_name

### PR DESCRIPTION
**Pull request by emichal**
And some code-sugar

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,4 +4,6 @@ python:
 install:
   - python setup.py -q install
 script:
-  - python setup.py -q test
+  - python setup.py -q ptr
+  - python setup.py -q ptr
```
Comment (mpaliwoda) [inline .travis.yml]: why `ptr` twice?
oslo: what is ptr?

File: flask_rage.py [hunk 2]
```diff
@@ -16,31 +18,33 @@
...
+        output["severity"] = record.levelname
+        output["message"] = record.getMessage()
 
         return json.dumps(output)
 
 
 class FlaskRage:
     """
-    Mimic RoR's lograge 'access log' formatting
+    Mimic RoR"s lograge "access log" formatting
```
Comment (mpaliwoda) [inline flask_rage.py]: search and replace gone wrong (SEE MEMES)

File: flask_rage.py [hunk 3]
```diff
@@ -83,43 +87,43 @@ def log_request(self, response):
         log_fn(message, extra=extra)
         return response
 
-    def log_exception(self, exception):
+    def log_exception(self, exception: Exception) -> None:
         """
         Log an exception in lograge-ish format
 
-        This can be called e.g. from flask's errorhandlers
+        This can be called e.g. from flask"s errorhandlers
```
Comment (mpaliwoda) [inline flask_rage.py]: up^

File: .travis.yml (hunk 1 above)
Comment (emichal) [reply .travis.yml]: > ptr               run unit tests after in-place build
I have no idea why stuff got duplicated. _Literally_

**Pull request merged**